azure-pipelines.yml

```diff
@@ -5,7 +5,7 @@ variables:
   # defined in UI
   #- ReleaseWheelBuild: False
   #- UploadWheel: False
-  buildWheel: ${{ or(in(variables['Build.Reason'], 'Schedule'), in(variables['Build.Reason'], 'Manual')) }}
+  buildWheel: True #TODO: revertthis ${{ or(in(variables['Build.Reason'], 'Schedule'), in(variables['Build.Reason'], 'Manual')) }}
```

Comment:
.

Reply:
Only 3.9 gets built for PRs..

Reply:
Sorry, right. I got confused by the micro-diff.

Reply:
_keeping this unresolved to avoid merging by mistake_